src/doc/gui_manual/Quantitative/LineSampler.rst

```diff
@@ -0,0 +1,331 @@
+.. _LineSampler:
+
+LineSampler
+-----------
+
+Similar to the Lineout tool one-dimensional curves, created using data
+from 2D or 3D plots, are popular for analyzing data because they are
+simple to compare. However, unlike the Lineout which creates a single
+curve it is a tool, whereas the LineSampler is an operator and can
+create multiple curves.
+
```

Comment:
This paragraph needs rewording.  If I am understanding the pertinent thoughts:
"one dimensional curves created using data from 2d or 3d plots are popular for analyzing data because they are simple to compare'
'LineSampler creates multiple curves, whereas Lineout creates a single curve.'
If that assessment is correct, perhaps something like this:
'One dimensional curves, created using data from 2D or 3D plots, are popular for analyzing data because they are simple to compare.  LineSampler is similar to Lineout in this regard except that it will create multiple curves instead of just a single curve.`